internal/it/util.go

```diff
@@ -458,3 +458,37 @@ func WaitEventuallyWithTimeout(t *testing.T, wg *sync.WaitGroup, timeout time.Du
 		t.FailNow()
 	}
 }
+
+// Cloud APIs
+
+func LoginToHazelcastCloudUsingEnvironment(ctx context.Context) error {
+	return rc.LoginToHazelcastCloudUsingEnvironment(ctx)
+}
+
+func LoginToHazelcastCloud(ctx context.Context, uri string, apiKey string, apiSecret string) error {
+	return rc.LoginToHazelcastCloud(ctx, uri, apiKey, apiSecret)
+}
+
+func CreateHazelcastCloudStandardCluster(ctx context.Context, hzVersion string, isTlsEnabled bool) (*CloudCluster, error) {
+	return rc.CreateHazelcastCloudStandardCluster(ctx, hzVersion, isTlsEnabled)
+}
+
+func GetHazelcastCloudCluster(ctx context.Context, clusterId string) (*CloudCluster, error) {
```

Comment:
Could you rename `clusterId` to `clusterID` due to Go naming conventions. Same comment for others.

Reply:
Updated